doc/source/parameters/cfd/physical_properties.rst

```diff
@@ -123,6 +124,15 @@ Physical Properties
 
 * The ``tracer reaction order`` parameter (:math:`n`) is the reaction order of the tracer.
 
+* The ``tracer reaction threshold`` parameter (:math:`\varepsilon`) is a minimal value set to prevent the explosion of stiffness for the linear solver when concentration tends toward 0. This parameter is useful when the reaction order is lower than 1.
```

Comment:
```suggestion
* The ``tracer reaction threshold`` parameter (:math:`\varepsilon`) is a minimal value set to prevent the explosion of stiffness for the linear solver when concentration is close to 0. This parameter is useful when the reaction order is lower than 1.
```

Reply:
I changed the explanation to just mention that when the reaction order < 1, a threshold value above 0 is required to prevent numerical issues. That feels honest but clear. What do you think?